frontend/src/pages/error.js

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
A generic route is just a .js file that exports a function which does `return inject(<MyComponent />)`